ui/imports/shared/stores/PermissionsStore.qml

```diff
@@ -17,10 +17,6 @@ QtObject {
     readonly property var permissionsModel:
         chatCommunitySectionModuleInst.permissionsModel
 
-    function setViewOnlyCanAddReaction(chatId, checked) {
-        //TODO: backend implementation
-    }
```

Comment:
I removed those because that field isn't updated independently, it's updated using the EditChat API